armada_jupyter/armada.py

```diff
@@ -24,7 +26,7 @@ def create_armada_request(job: Job, client: ArmadaClient):
     )
 
 
-def submit(submission: Submission):
+def submit(submission: Submission, job: Job, parent_client: ArmadaClient):
```

Comment:
Can you add typings to the armada_jupyter code if they have returns?

Reply:
Sure :))